### PR TITLE
feat(post): add netiquette agreement

### PR DESCRIPTION
**Pull request by yellowcap**
Depends on #3 .

@photonbit  and @JotaFan again invited to review 😁

### CODE REVIEW

**Review by JotaFan — APPROVED** [review]
🍸

File: _posts/2023-02-16-communications-netiquette.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Meetings should be scheduled in advance and agreed by all the assistants, specially if it is a one-on-one meeting.
+
+For ad-hoc meeting requests, the same rule apply by asking about availability over slack.  If a person is dedicating their full attention to another task, creating a zoom call before agreeing on starting the meeting creates a tension and a distraction and it must be avoided.
+
+The topic of the meeting should also be clarified in advance, and ideally also the agenda and desired outcomes.
+
+As a principle, we should avoid meetings where people disconnect because they are not involved. So if people feel disconnected or bored in a meeting, something about the scheduling went wrong.
+
+Meetings should be kept under 45 minutes to make it possible for people to stay focused. Scheduling meetings on Thursdays is discouraged because of [Focus Thursdays](https://bahamut.slab.com/posts/focus-thurs-day-qep21nok).
```
Comment (photonbit) [inline _posts/2023-02-16-communications-netiquette.md]: We will see this particular issue in other post coming from Bahamut, what should we do with the internal links to the wiki to documents that haven't been published? The current unmodified link will take us to the login of Bahamut, maybe we want to add a label or emoji that indicates that it is still not available or add a foot note to briefly explain what the document is about.

**Review by photonbit — COMMENTED** [review]
This thing about the link, otherwise looks beautiful

File: _posts/2023-02-16-communications-netiquette.md (hunk 1 above)
Comment (yellowcap) [reply _posts/2023-02-16-communications-netiquette.md]: Yes I am going back and fourth between simply publishing on slab vs integrating here. But I think integrating will be better, like this its all open source on github and in one place. So I will substitute it with a relative link and then we can add the focus thursdays one next.

**Pull request merged**